src/DocumentFormat.OpenXml.Framework/Equality/OpenXmlElementEqualityOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DocumentFormat.OpenXml
+{
+    /// <summary>
+    /// Options defining the behaviour of equality for <see cref="OpenXmlElement"/>.
+    /// </summary>
+    public sealed class OpenXmlElementEqualityOptions
+    {
+        /// <summary>
+        /// Gets a value indicating whether extended attributes should be considered when determining equality.
+        /// </summary>
+        public bool IncludeExtendedAttributes { get; init; } = true;
```

Comment:
inits will fail the CI build due to requiring PEVerify on .NET Framework builds for common scenarios. Means we can't use records and other types that use that.